src/tests/conftest.py

```diff
@@ -25,12 +25,18 @@ def forecasts(db_session):
 @pytest.fixture
 def db_connection():
     """Pytest fixture for a database connection"""
-    # with tempfile.NamedTemporaryFile(suffix="db") as tmp:
-    #     set url option to not check same thread, this solves an error seen in testing
 
-    # url = f"sqlite:///{tmp.name}.db?check_same_thread=False"
-    # os.environ["DB_URL"] = url
-    # os.environ["DB_URL_PV"] = url
+    # -- Uncomment for dockerised testing --
```

Comment:
should we use this all the time?

Reply:
Well I reckon so, however it didn't work in github actions when I tried it and didn't think it was worth the time investment to figure it out just yet. Odd thing is, the other method doesn't work locally for me...!